test/unittest/StatisticsReaderListener/StatisticsReaderListenerTests.cpp

```diff
@@ -388,20 +390,26 @@ TEST_F(statistics_reader_listener_tests, new_history_latency_received)
 TEST_F(statistics_reader_listener_tests, new_network_latency_received)
 {
     std::array<uint8_t, 16> src_locator_address = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16};
-    eprosima::fastrtps::rtps::Locator_t src_locator_t;
+    eprosima::fastdds::
+            rtps::Locator_t src_locator_t;
```

Comment:
Apply to remaining cases
```suggestion
    eprosima::fastdds::rtps::Locator_t src_locator_t;
```